CHANGES

```diff
@@ -13,6 +13,15 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" https://semver.org
 
 ------------------------------------------------------------------------------
+*/*/* xiki-tempula,
+
+  * unreleased
+
+Changes
```

Comment:
Is this really a change or shouldn't this be listed as a Fix because it now does what is advertised?

Reply:
versionchanged added